inst/stan/functions/generated_quantities.stan

```diff
@@ -14,7 +14,7 @@ int[] report_rng(vector reports, real[] rep_phi, int model_type) {
   int sampled_reports[t];
   if (model_type) {
     for (s in 1:t) {
-      sampled_reports[s] = neg_binomial_2_rng(reports[s] > 1e8 ? 1e8 : reports[s], rep_phi[model_type]);
+      sampled_reports[s] = neg_binomial_2_rng(reports[s] > 1e7 ? 1e7 : reports[s], rep_phi[model_type]);
```

Comment:
I don't think that the `reports` causes the overflows/exceptions and `1e8` is well bellow `.Machine$integer.max`. I believe that `rep_phi` is the culprit (see #147).